api/clever/__init__.py

```diff
@@ -251,32 +255,36 @@ def remote_patron_lookup(self, token):
 
         """
         bearer_headers = {'Authorization': 'Bearer %s' % token}
-        result = self._get(self.CLEVER_API_BASE_URL + '/me', bearer_headers)
+        result = self._get(self.CLEVER_API_VERSIONED_URL +
+                           '/me', bearer_headers)
         data = result.get('data', {}) or {}
 
         identifier = data.get('id', None)
 
         if not identifier:
             return INVALID_CREDENTIALS.detailed(lgt("A valid Clever login is required."))
 
-        if result.get('type') not in self.SUPPORTED_USER_TYPES:
+        if result.get('user_type') not in self.SUPPORTED_USER_TYPES:
```

Comment:
This looks great! Small question around this field getting renamed. I was looking at the [Clever API Explorer](https://dev.clever.com/reference/me) for this endpoint and it looks like this field is still named `type`, just want to make sure I'm looking at the right thing.

Reply:
Yeah, thanks for the catch.  I believe I was looking at some of the changes to the user model that do not affect us.  Changes reverted.